docs/serving/services/creating-private-services.md

```diff
@@ -0,0 +1,65 @@
+---
+title: "Creating a private Knative Service"
+linkTitle: "Creating a private Knative Service"
+weight: 20
+type: "docs"
+aliases: [../serving/cluster-local-route.md]
```

Comment:
does that syntax work? ive not seen it that way but i also dont know if that works

for reference, i know this syntax works: https://raw.githubusercontent.com/knative/docs/master/docs/serving/configuring-autoscaling.md